crates/anvil/src/eth/backend/db.rs

```diff
@@ -348,7 +348,7 @@ pub struct SerializableAccountRecord {
     pub nonce: u64,
     pub balance: U256,
     pub code: Bytes,
-    pub storage: BTreeMap<U256, U256>,
+    pub storage: BTreeMap<B256, B256>,
```

Comment:
this is also how the regular genesis account is formatted with hash -> hash

Reply:
I see an example of genesis.json storage field: 

```
      "storage": {
        "0x0000000000000000000000000000000000000000000000000000000000000001": "0x02",
        "0x0000000000000000000000000000000000000000000000000000000000000002": "0x04",
        "0x29ecdbdf95c7f6ceec92d6150c697aa14abeb0f8595dd58d808842ea237d8494": "0x01",
        "0x6aa118c6537572d8b515a9f9154be55a3377a8de7991cd23bf6e5ceb368688e3": "0x01",
        ...
      }
```

The type is `pub storage: BTreeMap<B256, U256>`, but it doesnt resolve the issue:
https://github.com/foundry-rs/foundry/issues/8179